components/expand-collapse/expand-collapse-content.js

```diff
@@ -44,8 +46,11 @@ class ExpandCollapseContent extends LitElement {
 
 			.d2l-expand-collapse-content-container {
 				display: none;
+				opacity: 0;
 				overflow: hidden;
-				transition: height 400ms cubic-bezier(0, 0.7, 0.5, 1);
+				transition:
+					height var(--d2l-expand-collapse-content-transition-duration) var(--d2l-expand-collapse-content-transition-function),
+					opacity var(--d2l-expand-collapse-content-transition-duration) var(--d2l-expand-collapse-content-transition-function);
```

Comment:
Maybe I missed this in another PR, but where is the `--d2l-expand-collapse-content-transition-function` defined? Doesn't come up on a search of core.